domain/port/state/state.go

```diff
@@ -98,7 +98,12 @@ func (st *State) GetMachineOpenedPorts(ctx context.Context, machine string) (map
 	machineUUID := machineUUID{UUID: machine}
 
 	query, err := st.Prepare(`
-SELECT &unitEndpointPortRange.*
+SELECT 
```

Comment:
Turn the common part of this query and the one you've changed below into a view.
Then you can select `&unitEndpointPortRange` from it again.

Reply:
Done